adoc/reference-webui-systems.adoc

```diff
@@ -2836,12 +2836,20 @@ endif::[]
 == Systems > Virtual Host Managers
 
 
-Third party hypervisors and hypervisor managers such as VMWare vCenter are called "`Virtual Host Managers`" (VHM) within {productname}.
-These managers can manage one or multiple virtual hosts, which in turn may contain virtual guests. {productname} ships with a tool called [command]``virtual-host-gatherer`` that can connect to VHMs using their API, and request information about virtual hosts.
-This tool is automatically invoked via Taskomatic nightly, therefore you need to configure your VHMs via XMLRPC APIs. [command]``virtual-host-gatherer`` maintains the concept of optional modules, where each module enables a specific Virtual Host Manager.
+Third party hypervisors and hypervisor managers such as VMWare vCenter are called "`Virtual Host Managers`" (VHM).
+VHMs can manage one or more virtual hosts, and each virtual host can contain one or more virtual guests.
+
+To create a VHM, navigate to menu:Systems[Virtual Host Managers] in the {webui}.
+In the upper right of the page, click btn:[Create] and select either [guimenu]``VMware-based`` or [guimenu]``File-based``.
+Continue according to one of the following procedures.
+
+After it has been created and configured, you can begin data collection from your VHMs manually through the {webui}, by navigating to menu:Systems[Virtual Host Managers], selecting the appropriate VHM, and clicking btn:[Refresh Data].
```

Comment:
Just to be extra clear - clicking on "Refresh Data" has the same effect of waiting a nightly Taskomatic round - it results in `virtual-host-gatherer` being run.